lib/feature/add_character/cubit/add_character_cubit.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:character_viewer/common/network/dto/character.dart';
+import 'package:character_viewer/common/service/character_service.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:injectable/injectable.dart';
+
+part 'add_character_cubit.freezed.dart';
+part 'add_character_state.dart';
+
+@injectable
+class AddCharacterCubit extends Cubit<AddCharacterState> {
+  final CharactersService _charactersService;
+
+  AddCharacterCubit(this._charactersService)
+      : super(const AddCharacterState.initial());
+
+  void addCharacter({
+    required String title,
+    required String imageUrl,
+    required String description,
+  }) {
+    try {
+      final newCharacter = Character(
+        title: title,
+        imageUrl: imageUrl,
+        description: description,
+      );
+
+      _charactersService.addCharacter(newCharacter);
```

Comment:
```suggestion
      await _charactersService.addCharacter(newCharacter);
```

Reply:
Need advice!))
If I have changed to 'async await' as a result the  snake bar doesn't work (